shaptools/hdb_connector/connectors/base_connector.py

```diff
@@ -34,6 +34,31 @@ class QueryError(BaseError):
     Error during query
     """
 
+class QueryResult(object):
+    """
+    Class to manage query results
+
+    Args:
```

Comment:
The args for the class docstring must talk about the `__init__` method parameters. We have to update that